src/components/Amalgamation/ForeignBusiness.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <section class="foreign-business">
+    <v-form
+      ref="foreignBusinessForm"
+      @submit.prevent
+    >
+      <v-row no-gutters>
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <label>Add a Foreign Business</label>
+        </v-col>
+        <v-col
+          cols="12"
+          sm="9"
+        >
+          <NestedSelect
+            label="Select the home jurisdiction"
+            :errorMessages="jurisdictionErrorMessage"
+            :menuItems="jurisdictionOptions"
+            :value="jurisdiction ? jurisdiction.text : ''"
+            @change="onJurisdictionChange($event)"
+          />
+        </v-col>
+        <v-col
+          cols="12"
+          sm="9"
+          offset="3"
+        >
+          <v-text-field
+            v-model="legalName"
+            filled
+            label="Business' full legal name in home jurisdiction"
+            :rules="rules"
+          />
+        </v-col>
+        <v-col
+          cols="12"
+          sm="9"
+          offset="3"
+        >
+          <v-text-field
+            v-model="corpNumber"
+            filled
+            label="Corporate number in home jurisdiction"
+            :rules="rules"
+          />
+        </v-col>
+        <v-col
+          cols="auto"
+          class="ms-auto"
+        >
+          <v-btn
+            large
+            color="primary"
+            class="mr-3"
+            @click="addForeignBusiness()"
+          >
+            Add
+          </v-btn>
+          <v-btn
+            large
+            outlined
+            color="primary"
+            @click="emitCancel()"
+          >
+            Cancel
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+  </section>
+</template>
+<script lang="ts">
```

Comment:
Add blank line between template and script block (and also between script and style block below).